fix(ui): Cosmetic changes to improve responsive translated views

Jeykairis opened the pull request:
## PR Checklist

<!--

Hi, thanks for taking the time to submit this Pull Request, it is really appreciated!

Before submitting the Pull Request make sure you:

* are familiar with and follow the Code of Conduct for
  this project: CODE_OF_CONDUCT.md

* the commit message follows our guidelines: CONTRIBUTING.md#commit-message-guidelines

Feel free to add you as a contributor via `npm run contributors:add`

-->

## What:
<!-- What changes are being made, why are these changes necessary? -->
Some cosmetic changes to make the starter looks better with every languages on every screen sizes.
<!--  To automatically close the corresponding issue use: Closes #<issue-number, e.g. Closes #47 -->
Issue number: N/A

tomastrajan commented on src/app/examples/examples/examples.component.scss:
```diff
@@ -3,6 +3,7 @@
 nav {
   margin-bottom: 20px;
   position: relative;
+  overflow: scroll;
```
This seems to be unfortunately  broken in Google Chrome...
![anms-error](https://user-images.githubusercontent.com/3764868/45807826-9b4e1e00-bcc4-11e8-8825-7ea7b39caa32.png)

tomastrajan commented:
@Jeykairis also what could be worth investigating is to make that sub-menu responsive in a sense that on small screens it will be replaced by the dropdown (select) element, thoughts ?

tomastrajan commented on src/index.html:
```diff
@@ -71,7 +76,7 @@
   <anms-root>
     <div class="app-loading">
-      <div class="logo"></div>
+      <img src="./assets/logo.png">
```
This will not work for the prod build, that was the reason to extract this to css because that way it is processed by the Weback as an asset because the `styles-app-loading.css` is referenced in the `angular.json` as a external stylesheet. Please revert this change.

tomastrajan replied:
@Jeykairis hmm, looks like this was broken when migrating to Angular CLI 6+ with new `angular.json` format because as it seems the `styles-app-loading.scss` is not referenced anywhere in the whole application... The correct way would be to add it to the `styles` in the `angualr.json`. Another thing worth trying would be to just add the `logo.png` into the `assets` in `angular.json` but then it is the question what would happen during the prod build (eg you don't get the content hashing and cache busting) so maybe it is still better to stick to the `styles-app-loading.scss` which generates hashed asset (eg `logo-24rfsdgd23r.png`).

Jeykairis replied:
Okay, thank you for the explanations!
I tried to import the `styles-app-loading.scss` into the `styles.scss` and the logo appears but not the spinner...
I try to add the `styles-app-loading.scss` directly in the `angular.json`.

tomastrajan replied:
@Jeykairis the way this behaves is that the spinner should be shown immediately because it is the part of initial `index.html` template, after that when the styles bundle is loaded for `styles-app-loading.scss` the logo should appear and after that when the whole application is loaded it will replace the spinner and the logo with the app itself ;)

Jeykairis commented:
Haha, it was looking good for chrome on a mac but it was a dirty fix. I cleaned it and implemented the select for smallest screens.
By the way, we should take care of the material specs regarding the sub-nav https://material.io/design/components/tabs.html#behavior.

Thanks @timdeschryver for fixing the autosave !
